lib/miq_automation_engine/service_models/miq_ae_service_storage_profile.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceStorageProfile < MiqAeServiceModelBase
+  end
```

Comment:
Just `:storages`.  The `expose` command in automate ends up calling the method on the active record object.